runtimes/pythonrt/ak_runner.py

```diff
@@ -0,0 +1,343 @@
+"""Run user code under AutoKitteh"""
+
+# This file is long, but keeping it a single file helps with embedding it in Go and
+# running it.
```

Comment:
But Go can embed an entire directory with `*.py`, no?

Anyway, a cosmetic suggestion:

```suggestion
"""Run user code under AutoKitteh.

This module is long, but keeping everything in a single file helps with
embedding it in Go and running it.
"""
```

Reply:
We can embed a file system, but it makes the extraction code a bit more complex. I think for now it's OK to keep it in a single file, and once the size/complexity becomes an issue - split it. 
Also, there might be issues with imports colliding with user modules with the same way (can overcode with relative imports, but again - complexity).